src/main/java/graphql/DeprecatedAt.java

```diff
@@ -0,0 +1,22 @@
+package graphql;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.TYPE;
+
+/**
+ * Helper to track deprecation
+ *
+ * Please use ISO-8601 format i.e. YYYY-MM-DD
+ */
+@Retention(RetentionPolicy.SOURCE)
```

Comment:
I picked `SOURCE` level, as this annotation is only to help contributors. Open to ideas if you want another retention policy